.circleci/config.yml

```diff
@@ -61,10 +61,13 @@ jobs:
           - $HOME/go/src/
 
       # Run pre-commit hooks and fail the build if any hook finds required changes.
-      - run: go get golang.org/x/tools/cmd/goimports
-      - run: pip install pre-commit==1.11.2
-      - run: pre-commit install
-      - run: pre-commit run --all-files
+      - run:
+          name: run precommit
+          command: |
+            go get golang.org/x/tools/cmd/goimports
+            pip install pre-commit==1.11.2 cfgv==2.0.1
```

Comment:
NIT: add a comment about why we need that specific version of cfgv

Reply:
:+1: Shamelessly copying from `terraform-aws-couchbase` 0d4d791